UI/src/router.js

```diff
@@ -203,9 +203,10 @@
         'lengthLimit/:id': 'lengthLimit',
         'widthLimit/:id': 'widthLimit',
         'manoeuvres/:id': 'manoeuvres',
+        'parkingProhibition/:id': 'parkingProhibition',
         'work-list/speedLimit(/:administrativeClass)': 'speedLimitWorkList',
         'work-list/speedLimit/municipality(/:id)': 'speedLimitMunicipalitiesWorkList',
-        'work-list/speedLimitErrors(/:administrativeClass)': 'speedLimitErrorsWorkList',
+        'work-list/speedLimitErrors': 'speedLimitErrorsWorkList',
```

Comment:
check this difference, doesn't make sense have this difference in that branch

Reply:
I have doubts in this onde. Could it be a merge during the development of the US?